packages/main/src/plugin/index.ts

```diff
@@ -181,7 +181,7 @@ export class PluginSystem {
   private uiReady = false;
 
   // true if the application is quitting
-  private isQuitting = false;
+  public isQuitting = false;
```

Comment:
I think that it needs to stay private

if you need the value, you can add an accessor but we can't let anyone being able to override the value